SPECS/popt/popt.spec

```diff
@@ -1,14 +1,13 @@
 Summary:	Programs to parse command-line options
 Name:		popt
-Version:	1.16
-Release:        7%{?dist}
+Version:	1.18
+Release:    1%{?dist}
 License:	MIT
-URL:		http://rpm5.org/files/popt
+URL:		https://github.com/rpm-software-management/popt
 Group:		Applications/System
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:	ftp://anduin.linuxfromscratch.org/BLFS/svn/p/%{name}-%{version}.tar.gz
-%define sha1 popt=cfe94a15a2404db85858a81ff8de27c8ff3e235e
+Source0:	http://ftp.rpm.org/popt/releases/popt-1.x/%{name}-%{version}.tar.gz
```

Comment:
Nit: https

Reply:
actually the https protocol is not supported by that server :(

Reply:
and github release recommend to download source tarball from that http server :( :(